src/translations/sqlb_nl.ts

```diff
@@ -0,0 +1,7344 @@
+        <source>&amp;SQL</source>
+        <translation>&amp;SQL</translation>
+    </message>
+    <message>
+        <location filename="../PreferencesDialog.ui" line="1203"/>
+        <source>Settings name</source>
+        <translation>Instellingsnaam</translation>
+    </message>
+    <message>
+        <location filename="../PreferencesDialog.ui" line="1208"/>
+        <source>Context</source>
+        <translation>Context</translation>
+    </message>
+    <message>
+        <location filename="../PreferencesDialog.ui" line="1213"/>
+        <source>Colour</source>
+        <translation>Kleur</translation>
+    </message>
+    <message>
+        <location filename="../PreferencesDialog.ui" line="1218"/>
+        <source>Bold</source>
+        <translation>Vet</translation>
+    </message>
+    <message>
+        <location filename="../PreferencesDialog.ui" line="1223"/>
+        <source>Italic</source>
+        <translation>Cursief</translation>
+    </message>
+    <message>
+        <location filename="../PreferencesDialog.ui" line="1228"/>
+        <source>Underline</source>
+        <translation>Onderstreept</translation>
+    </message>
+    <message>
+        <location filename="../PreferencesDialog.ui" line="1236"/>
+        <source>Keyword</source>
+        <translation>Sleutelwoord</translation>
+    </message>
+    <message>
+        <location filename="../PreferencesDialog.ui" line="1256"/>
+        <source>Function</source>
+        <translation>Functie</translation>
+    </message>
+    <message>
+        <location filename="../PreferencesDialog.ui" line="1264"/>
+        <source>Table</source>
+        <translation>Tabel</translation>
+    </message>
+    <message>
+        <location filename="../PreferencesDialog.ui" line="1284"/>
+        <source>Comment</source>
+        <translation>Commentaar</translation>
+    </message>
+    <message>
+        <location filename="../PreferencesDialog.ui" line="1304"/>
+        <source>Identifier</source>
+        <translation>Entiteitsnaam</translation>
+    </message>
+    <message>
+        <location filename="../PreferencesDialog.ui" line="1324"/>
+        <source>String</source>
+        <translation>Tekenreeks</translation>
+    </message>
+    <message>
+        <location filename="../PreferencesDialog.ui" line="1344"/>
+        <source>Current line</source>
+        <translation>Huidige regel</translation>
+    </message>
+    <message>
+        <location filename="../PreferencesDialog.ui" line="1352"/>
+        <source>Background</source>
+        <translation>Achtergrond</translation>
+    </message>
+    <message>
+        <location filename="../PreferencesDialog.ui" line="1360"/>
+        <source>Foreground</source>
+        <translation>Voorgrond</translation>
+    </message>
+    <message>
+        <location filename="../PreferencesDialog.ui" line="1365"/>
+        <source>selected_bg</source>
+        <translation></translation>
```

Comment:
I think it's a proper noun that doesn't need translation, so I think you've left the translation blank.
I also did like this in the past, but there was an comments that it is safer to copy the original text for proper nouns
that do not require translation. Please refer to [here](https://github.com/sqlitebrowser/sqlitebrowser/pull/2329#issuecomment-658202890) for details.

Reply:
I thought this might have been a variable name, that erroneously got submitted for translation, but I've copied the original now.